config/database.sql

```diff
@@ -0,0 +1,15 @@
+CREATE DATABASE users;
```

Comment:
Could u create another sql file to load data please

Reply:
done